File: LeetCode/1748.py
#Link: https://leetcode.com/problems/sum-of-unique-elements/

class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        total = 0
        for num in nums:
            if nums.count(num) == 1:
                total += num
        return total
        

File: LeetCode/cpp/1748.cpp
//Link: https://leetcode.com/problems/sum-of-unique-elements/

class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        int counts[101] = {0}, sum = 0;
        for(auto num : nums)
            counts[num]++;
        for(int i = 0 ; i < 101 ; i++)
            if(counts[i] == 1)
                sum += i;
        return sum;
    }
};
